src/applications/pensions/components/HomeAcreageValueInput.jsx

```diff
@@ -84,7 +90,18 @@ const HomeAcreageValueInput = props => {
   };
 
   const navButtons = <FormNavButtons goBack={goBack} submitToContinue />;
-  const updateButton = <va-button type="submit">Update</va-button>;
+  const updateButton = (
+    <div className="form-nav-buttons vads-u-margin-top--4">
+      <va-button
+        onClick={e => {
+          updateFormData(e);
+          updatePage(e);
+        }}
+        label="Update"
+        text="Update"
+      />
+    </div>
+  );
```

Comment:
This 'Update' button is not full width like the other 'Update' buttons within the 'Review application' page.